ui/react-components/components/EditAppointment/EditAppointment.jsx

```diff
@@ -254,11 +341,24 @@ const EditAppointment = props => {
         defaultTime: appointmentDetails.endTime
     };
 
+    const isStartDateModified = () => !isEqual(moment(originalAppointmentDate).format('DMYYYY'), moment(appointmentDetails.appointmentDate).format('DMYYYY'));
+
+    const isEndDateModified = () => !isEqual(moment(originalRecurringEndDate).format('DMYYYY'), moment(appointmentDetails.recurringEndDate).format('DMYYYY'));
+
+    const isOccurrencesModified = () => originalOccurrences !== appointmentDetails.occurrences
+
+    const isApplicableForAll = () => {
+        if (isStartDateModified())
+            return false;
+        else if (isEndDateModified() || isOccurrencesModified())
+            return false;
```

Comment:
The following looks good? `return !(isStartDateModified() || isEndDateModified() || isOccurrencesModified())`

Reply:
DONE